src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -1069,3 +1077,62 @@ export const disableSmsVerificationsForUser = (
       return transformMongoError(error)
     },
   ).map(() => true)
+
+/**
+ * Checks the number of free smses sent by the admin of the form and deactivates verification or sends mail as required
+ * @param form The form whose admin's sms counts needs to be checked
+ * @returns ok(true) when the verification has been deactivated successfully or no action is required
+ * @returns err(MailGenerationError) when an error occurred on creating the HTML template for the email
+ * @returns err(MailSendError) when an error occurred on sending the email
+ * @returns err(PossibleDatabaseError) when an error occurred while retrieving the counts from the database
+ */
+export const checkFreeSmsSentByAdminAndDeactivateVerification = (
```

Comment:
Maybe move this into VerificationService since this is only used when verification controller calls this fn

I know this is probably following the deactivate form function signature but this does not really need to be in a single function (just like deactivate form does not need to 😢 ). 
Maybe we can split this into two functions and have the (only) controller call in VerificationController call them both in order.

Reply:
Rereading the code, it does not even deactivate the form, only possibly deactivate the form. So the function name is already a misnomer :P.

Idk name suggestions sorry

Reply:
wrt this, per what we discussed irl, will change the naming but keep the same behaviour for neatness! thinking of `processAdminSmsCounts`, wdyt?

Reply:
agree